.github/scripts/generate_dataset_table.py

```
@@ -0,0 +1,65 @@
+
+TASKS_DIR = Path("lm_eval/tasks")
+OUTPUT_FILE = Path("docs/dataset-mapping.md")
+
+rows = []
+
+for yaml_file in sorted(TASKS_DIR.rglob("*.yaml")):
+    try:
+        with open(yaml_file) as f:
+            data = yaml.safe_load(f)
+    except Exception:
+        continue
```

Ruff (0.15.12)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)